README.md

```diff
@@ -17,21 +17,26 @@ Turn is a turn-based terminal RPG game written in C++. Contributions are encoura
 Clone this project by running `git clone https://github.com/tagniam/Turn.git` in your terminal.
 
 ### Building & Running
-#### Windows
-* Run CMake on the `Turn` directory to generate a `.sln` solution file for Visual Studio. You can find a tutorial [here](https://cmake.org/runningcmake/).
-* Run Visual Studio and open the generated `.sln` solution file.
-* Build the project by clicking `Build` -> `Build Solution`.
-* Start the game by clicking the green Run button in the toolbar.
-
 #### Mac/Linux
 * In your terminal, make sure you are in the `Turn` directory.
 * Run the following commands:
 
     ```shell
-    $ cmake .
-    $ make
+    $ cmake -S . -B build
+    $ cmkae --build build
+    ```
+* Start the game by running `./build/Turn`.
+
+#### Windows
+* In your terminal, make sure you are in the `Turn` directory.
+* For the right generator see [Visual Studio Generators](https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#visual-studio-generators)
+* To build with Visual Studio 16 2019 run the following commands:
+
+    ```shell
+    $ cmake -S . -B build -G "Visual Studio 15 2017"
+    $ cmkae --build build
     ```
-* Start the game by running `./Turn`.
+* Start the game by running `build\Turn.exe`.
```

Comment:
Are there instructions for the CMake GUI as well? Lots of contributors usually use the GUI to generate the VS project files rather than through command line.

Reply:
Also, please reorder the section so that Windows comes before Mac/Linux, most contributors use Windows so I'd like that to be the first thing that comes up (though I use Linux myself :))

Reply:
yes, building in VS GUI with the generated solution file is still possible, added it as `Alternative`

Reordered Windows back up